hardhat.config.ts

```diff
@@ -468,6 +468,9 @@ const config: HardhatUserConfig = {
     outDir: "./typechain",
     target: "ethers-v5",
   },
+  paths: {
+    tests: "./test/evm/hardhat",
```

Comment:
You've double checked that this still runs all the tests, right? Just want to double check that a change like this doesn't accidentally disable all the evm tests because of a minor typo or config issue causes it to point at an empty dir.

Reply:
![2024-09-15 at 19 19 08@2x](https://github.com/user-attachments/assets/1692ca9a-778a-4e44-9f7d-c4b5c85e0a4a)

Reply:
![2024-09-15 at 19 19 43@2x](https://github.com/user-attachments/assets/ca094969-2e21-4d9c-8f0a-d1728724535b)

Reply:
Amazing